packages/rfw/README.md

```diff
@@ -208,7 +345,580 @@ class _ExampleState extends State<Example> {
+closure that reports an event as described above, the
+`DataSource.handler` method is used.
+
+In addition to the list of keys that identify the `event` value, the
+method itself takes a callback closure. That callback's purpose is to
+convert the given `trigger` (a function which, when called, reports
+the event) into the kind of callback closure the `Widget` expects.
+This is usually written something like the following:
+
+<?code-excerpt "test/readme_test.dart (onTapDown)"?>
+```dart
+return GestureDetector(
+  onTapDown: source.handler(<Object>['onTapDown'], (HandlerTrigger trigger) => (TapDownDetails details) => trigger()),
+  child: source.optionalChild(<Object>['child']),
+);
+```
+
+To break this down more clearly:
+
+<?code-excerpt "test/readme_test.dart (onTapDown-long)"?>
+```dart
+return GestureDetector(
+  // onTapDown expects a function that takes a TapDownDetails
+  onTapDown: source.handler<GestureTapDownCallback>( // this returns a function that takes a TapDownDetails
+    <Object>['onTapDown'],
+    (HandlerTrigger trigger) { // "trigger" is the function that will send the event to RemoteWidget.onEvent
+      return (TapDownDetails details) { // this is the function that is returned by handler() above
+        trigger(); // the function calls "trigger"
+      };
+    },
+  ),
+  child: source.optionalChild(<Object>['child']),
+);
+```
+
+In some cases, the arguments sent to the callback (the
+`TapDownDetails` in this case) are useful and should be passed to the
+`RemoteWidget.onEvent` as part of its arguments. This can be done by
+passing some values to the `trigger` method, as in:
+
+<?code-excerpt "test/readme_test.dart (onTapDown-position)"?>
+```dart
+return GestureDetector(
+  onTapDown: source.handler(<Object>['onTapDown'], (HandlerTrigger trigger) {
+    return (TapDownDetails details) => trigger(<String, Object>{
+      'x': details.globalPosition.dx,
+      'y': details.globalPosition.dy,
+    });
+  }),
+  child: source.optionalChild(<Object>['child']),
+);
+```
+
+Any arguments in the `event` get merged with the arguments passed to
+the trigger.
+
+#### Animations
+
+The `rfw` package introduces a new Flutter widget called
+[`AnimationDefaults`](https://pub.dev/documentation/rfw/latest/rfw/AnimationDefaults-class.html).
+
+This widget is exposed by `createCoreWidgets` under the same name, and
+can be exposed in other local widget libraries as desired. This allows
+remote widget libraries to configure the animation speed and curves of
+entire subtrees more conveniently than repeating the details for each
+widget.
+
+To support this widget, implement curve arguments using
+[`ArgumentDecoders.curve`](https://pub.dev/documentation/rfw/latest/rfw/ArgumentDecoders/curve.html)
+and duration arguments using
+[`ArgumentDecoders.duration`](https://pub.dev/documentation/rfw/latest/rfw/ArgumentDecoders/duration.html).
+This automatically defers to the defaults provided by
+[`AnimationDefaults`]. Alternatively, the
```

Comment:
nit:
```suggestion
`AnimationDefaults`. Alternatively, the
```
(or make it a link)

Reply:
fixed (no link, link is already there earlier in the section)